lib/galaxy/config/sample/galaxy.yml.sample

```
@@ -64,12 +67,22 @@ gravity:
     # extra_args:
 
     # Use Gunicorn's --preload option to fork workers after loading the Galaxy Application.
-    # Consumes less memory when multiple processes are configured.
-    # preload: true
+    # Consumes less memory when multiple processes are configured. Default is ``false`` if using unicornherder, else ``true``.
+    # preload:
+
+    # Extra environment variables and their values to set when running the service. A dictionary where keys are the variable
+    # names.
+    # environment: []
```

Isn't `{}` the syntax for an empty dictionary in YAML?

```suggestion
    # environment: {}
```

Also below.

---

Yes, it looks like [I messed up the default values](https://github.com/galaxyproject/gravity/blob/8efd9c66b0efd3acb31fe153407d3ea80bf99e95/gravity/settings.py#L115), so I will need to correct it there, which I guess means cutting another release.

---

galaxyproject/gravity#58